cli/cmd/sources.go

```diff
@@ -67,6 +72,45 @@ odigos sources delete --group mygroup --all-namespaces
 	`,
 }
 
+var kindAliases = map[k8sconsts.WorkloadKind][]string{
+	k8sconsts.WorkloadKindDeployment:  []string{"deploy", "deployments", "deploy.apps", "deployment.apps", "deployments.apps"},
+	k8sconsts.WorkloadKindDaemonSet:   []string{"ds", "daemonsets", "ds.apps", "daemonset.apps", "daemonsets.apps"},
+	k8sconsts.WorkloadKindStatefulSet: []string{"sts", "statefulsets", "sts.apps", "statefulset.apps", "statefulsets.apps"},
+	k8sconsts.WorkloadKindNamespace:   []string{"ns", "namespaces"},
+}
+
+var sourceDisableCmd = &cobra.Command{
+	Use:   "disable [workload type] [workload name] [flags]",
+	Short: "Disable a source for Odigos instrumentation.",
+	Long:  "This command disables the given workload for Odigos instrumentation. It will create a Source object (if one does not already exist)",
+	Example: `
+# Disable deployment "foo" in namespace "default"
+odigos sources disable deployment foo
+
+# Disable namespace "bar" in namespace "default"
+odigos sources disable namespace bar
+
+# Disable statefulset "foo" in namespace "bar"
+odigos sources disable statefulset foo -n bar
+`,
+}
+
+var sourceEnableCmd = &cobra.Command{
+	Use:   "enable [workload type] [workload name] [flags]",
+	Short: "Enable a source for Odigos instrumentation.",
+	Long:  "This command enables the given workload for Odigos instrumentation. It will create a Source object (if one does not already exist)",
+	Example: `
```

Comment:
Not sure if it works in cobra for non-leaf commands, but wondering if we can have an `Alias` here to also react to `instrument` (and `uninstrument` for disable).